modules/form-field/react/stories/stories_Checkbox.tsx

```diff
@@ -171,7 +171,12 @@ const CheckboxStates = () => (
 );
 
 storiesOf('Components|Inputs/Checkbox/React/Visual Testing', module)
-  .addParameters({component: Checkbox})
+  .addParameters({
+    component: Checkbox,
```

Comment:
Interesting. It seems like Storybook is able to get the component just fine without this, but I'm not sure why.